packages/feedback-service/src/resolver.ts

```diff
@@ -0,0 +1,83 @@
+import { Feedback } from './schema';
+import { getTransporter } from './helpers';
+  
+const transporter = getTransporter;
+
+export const FeedbackResolver = {
+  Query: {
+    //queries
+    listFeedback(root: any, args: any, ctx: any) {
+      return Feedback.find()
+        .sort({ "timestamp": -1 })
+        .then(response => response)
+        .catch(err => err);
+    },
+    getFeedback(root: any, args: any, ctx: any) {
+      return Feedback.findById(args._id)
+        .then(response => response)
+        .catch(err => err);
+    }
+  },
+  Mutation: {
+    // mutations
+    addFeedback(root: any, args: any, ctx: any) {
```

Comment:
Refactor this resolver to remove the email logic. It is not needed, as JIRA will take care of that.
And even after that, yo can use the Notifications Microservice to send out the emails.